packages/api/src/functions/graphql.ts

```diff
@@ -12,6 +12,8 @@ export type GetCurrentUser = (
   raw: AuthContextPayload[1]
 ) => Promise<null | object | string>
 
+export type HasRole = (role: string) => Promise<boolean>
```

Comment:
Is this `HasRole` stuff still relevant? I thought you're overloading `requireAuth` as `requireAuth({ role })

Reply:
:) I removed this from my app's implementation ... but forgot to here. 

It is not needed.

That's what happens when I try to code at 3am.

I'll do a full walkthrough my next push.